packages/react-core/src/components/Alert/examples/Alert.md

```diff
@@ -71,11 +98,142 @@ import { Alert, AlertActionCloseButton, AlertActionLink } from '@patternfly/reac
   </React.Fragment>
 ```
 
-### Inline types
+#### Expandable alerts
+
+An alert can contain additional, hidden information that is made visible by clicking a chevron icon. This information can be expanded and collapsed each time the icon is clicked.
```

Comment:
We've generally referred to this icon as a "caret," not a "chevron." My understanding was always that a chevron was a double arrow.